app/lib/logic/rolling_updates.rb

```diff
@@ -250,16 +250,6 @@ def missing_config
         end
       end
 
-      class ApicastV1 < Base
```

Comment:
As I said earlier, @hallelujah suggested requested to deprecate rolling updates before removing them. See 
https://github.com/3scale/porta/pull/2496/files#diff-11abcaa744f5e2710aee0d37a5cce03dda18ef2d0a91312f6f5bb5843cae50f8R281

Reply:
As @hery said this class is no more used, removing the class itself